frontend/src/libs/components/input/styles.module.scss

```diff
@@ -61,3 +61,13 @@
 .search-input {
   padding-left: 30px;
 }
+
+.textarea {
+  width: 529px;
+  height: 225px;
```

Comment:
```suggestion
```
width should be 100%

height should be by rows

apply the same styles as for the input